docs/commands.md

```diff
@@ -0,0 +1,167 @@
+# Commands
+
+The Nautilus Librarian console commands.
+
+## Gold Images Processing
+
+It is a command to handle `Gold` image changes in an image dataset (we call it "Library").
+
+### Description
+
+This command allows you to process all the changes in a library repository affecting `Gold` images.
+
+Sample usage:
+
+```shell
+nautilus-librarian gold-images-processing \
+    --previous-ref PREVIOUS_REF \
+    --current-ref  CURRENT_REF
+```
+
+Where `PREVIOUS_REF` and `CURRENT_REF` ar  the git first and last commit references defining the list of commits you want to process.
+
+If you handle a dataset of images following the [Nautilus Filename Specification](https://github.com/Nautilus-Cyberneering/nautilus-namecodes), there is an special type of image called "Gold" image. That is the first artifact after the acquired media, for example via scanning.
```

Comment:
"there is **an** special type of"
You only add a "N" when the next word starts with a vocal.